packages/transitive-overlay/package.json

```diff
@@ -11,24 +11,28 @@
     "url": "https://github.com/opentripplanner/otp-ui/issues"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "tsc": "tsc"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/opentripplanner/otp-ui.git"
   },
   "dependencies": {
+    "@mapbox/polyline": "^1.1.1",
+    "@opentripplanner/base-map": "^3.0.0-alpha.3",
     "@opentripplanner/itinerary-body": "^4.0.0",
     "@opentripplanner/core-utils": "^7.0.0",
     "lodash.isequal": "^4.5.0",
-    "transitive-js": "^0.14.1"
+    "transitive-js": "^0.14.1",
```

Comment:
Is this still actually used?

Reply:
Good catch! I was being optimistic for the future but yes at the moment this is unused. I've removed it in 126e4300e171b9c3f7b1da89a3c885e544279db4

Reply:
Thanks! That will definitely cut down the bundle size and by the time we re-add transitive hopefully it will be much smaller, i.e. w/o the d3 dependency, etc.